### ISSUE TITLE
Can't get node to show incoming STRING

### ISSUE THREAD

**Issue by tom-doerr** [Suzie1/ComfyUI_Guide_To_Making_Custom_Nodes#1]
I am trying to crate a node that displays the incoming STRING, similar to how it is shown in the README. Would be helpful to me to have that code as well or link to it.

**Comment by tom-doerr** [comment]
Seems to be this node that is used: https://github.com/pythongosssss/ComfyUI-Custom-Scripts/blob/edc5321bfd3d8f77cace0a69b46e129d99b53c3d/py/show_text.py#L4

**Comment by schoenid** [comment]
> Seems to be this node that is used: https://github.com/pythongosssss/ComfyUI-Custom-Scripts/blob/edc5321bfd3d8f77cace0a69b46e129d99b53c3d/py/show_text.py#L4

Yes, or even `Display Any` (rgthree) or `Display Any` (ComfyUI Essentials) ...

It's a miracle, how to get a value into this text input or even on the blank surface of the nodes box.

**Comment by schoenid** [comment]
@tom-doerr Please have a look at https://github.com/Suzie1/ComfyUI_Guide_To_Making_Custom_Nodes/issues/2
Maybe this can solve it.
